data/INFC/data.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Infected",
+  "symbol": "INFC",
+  "decimals": 18,
+  "description": "Reward token in the Plague ecosystem",
+  "website": "",
```

Comment:
validation still failing because of the empty website strings here.   Do you have a website or some social media account you can link to here?

Reply:
Ah no sorry should I remove that line?